instrumentation/servlet/servlet-rw/src/test/java/io/opentelemetry/javaagent/instrumentation/hypertrace/servlet/rw/writer/PrintWriterInstrumentationTest.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright The Hypertrace Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.javaagent.instrumentation.hypertrace.servlet.rw.writer;
+
+import java.io.CharArrayWriter;
+import java.io.PrintWriter;
+import org.ServletReadWriteContextAccess;
+import org.TestPrintWriter;
+import org.hypertrace.agent.core.instrumentation.buffer.BoundedBuffersFactory;
+import org.hypertrace.agent.core.instrumentation.buffer.BoundedCharArrayWriter;
+import org.hypertrace.agent.testing.AbstractInstrumenterTest;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class PrintWriterInstrumentationTest extends AbstractInstrumenterTest {
+
+  private static final String BODY = "boobar";
+
+  @Test
+  public void write_single_char() {
+    PrintWriter printWriter = new TestPrintWriter(new CharArrayWriter());
```

Comment:
why do we need this class?

Reply:
`TestPrintWriter`

Reply:
We do no need this class. It is just to test if the instrumentation works for subclasses of PrintWriter as well